### ISSUE TITLE
대회 관련 문의드립니다.

### ISSUE THREAD

**Issue by yt-oh96** [DatathonInfo/MISOChallenge-pet#11]
1. 아래의 질문을 보니 pretrain 모델을 사용하여도 된다하였는데요. data에 대한 제약은 없는건가요? open된 데이터가 아닌 데이터로도 학습이 가능하다면, 데이터에서부터 평등하지 못한 대회가 될 것 같은데요. 제가 이해한대로 어떤 데이터로든 pretrain이 진행되어도 괜찮은게 맞나요?

2. 아래의 답변들을 보면 각 질환에 대하여 모델을 만들라고하셨는데요. 각 질환별로 모델을 만들어야하는게 규칙인건가요? 아니면 multi-label 모델을 만들어도 되는건가요? 솔루션에도 제약사항이 있는지 궁금하여 문의드립니다.

감사합니다.

**Comment by yt-oh96** [comment]
또한 한가지 더 문의를 드리자면, baseline을 제공해주셨는데 꼭 baseline에 맞춰서 해야하나요? (pytorch로 새로 구현하여 진행하여도 되나요?)

**Comment by DatathonInfo** [comment]
안녕하세요
yt-oh96 님

질문 주신 pre-trained 및 규칙,  pytorch 구현 관련 확인결과,

1. pre-trained 모델은 ImageNet으로 학습된 모델 weight를 이용한 전이학습의 경우에만 허용됩니다. 제공된 이미지 데이터 외의 데이터는 학습에 이용하실 수 없습니다. 

2. 백내장을 제외한 세가지 안구질환은 각각의 binary classification 모델을 제출해주셔야 합니다. 백내장과 피부질환의 경우에는 각각 multi-class classification 모델을 제출하셔야 합니다.

3. (추가질문) baseline 코드는 참가자님의 이해를 높이기 위해 제공된 참고용 코드이며, 실제 제출하실 모델은 Pytorch와 Tensorflow 중 어떤 프레임워크를 사용하셔도 무방합니다.
pytorch를 사용하실 경우 weight 파일은 pth로 제출하시면 됩니다.

감사합니다.
2021 미소 인공지능 모델 개발 첼린지 운영팀
